### PR TITLE
Check that url is set before trying to retrieve it

### PR DESCRIPTION
**Pull request by golenkovm**
This is a fix for #4

### CODE REVIEW

File: classes/helper.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ public static function redirect_from_rules() {
         $rules = self::get_all_rules();
 
         foreach ($rules as $rule) {
-            if ($rule->is_enabled() && $rule->should_redirect($PAGE->url)) {
+            if ($rule->is_enabled() && $PAGE->has_set_url() && $rule->should_redirect($PAGE->url)) {
```
Comment (dmitriim) [inline classes/helper.php:107]: Should we check that very first like 

`if ($PAGE->has_set_url()) {
}
`

**Comment by dmitriim** [comment]
@golenkovm would you be able to fix travis as part of this fix? I guess we can drop not supported versions

**Review by dmitriim — APPROVED** [review]

**Comment by dmitriim** [comment]
Let's wait for travis.

**Comment by golenkovm** [comment]
Travis is happy with this change \o/

**Pull request merged**